lib/views/frontend/spree/checkout/payment/_applepay.html.erb

```diff
@@ -0,0 +1,77 @@
+<span id="apple-pay-button"></span>
+
+<script type="text/javascript">
+
+  if (typeof Stripe === 'undefined') {
+    $.getScript('https://js.stripe.com/v2/', applePay);
+  } else {
+    applePay();
+  };
+
+  function applePay() {
+    <% stripe_payment_method = Spree::PaymentMethod.find_by(type: 'Spree::Gateway::StripeGateway') %>
+    <% unless (stripe_payment_method && stripe_payment_method.active? && stripe_payment_method.preferences[:publishable_key]).present? %>
```

Comment:
I think we should only check it for typeof Stripe === 'undefined', as there might be a case when stripe is active and present in the system, but will be initialized after PayPal, so in that case here we will not set publishable key, which would be wrong.